docs/source/en/_toctree.yml

```diff
@@ -101,6 +101,8 @@
       title: Instantiating a big model
     - local: debugging
       title: Debugging
+    - local: hpo_train
+      title: Hyperparameter Search using Trainer API
```

Comment:
Don't think this PR should touch this

Reply:
Yes, I copy pasted the latest main file, merging with main should remove the modification